packages/editor-ui/src/components/RunData.vue

```diff
@@ -93,10 +94,11 @@
 				</el-dropdown>
 				<n8n-icon-button
 					:title="$locale.baseText('runData.editOutput')"
+					:circle="false"
+					class="ml-2xs"
 					icon="pencil-alt"
 					type="tertiary"
-					:circle="false"
-					@click="enterEditMode"
+					@click="enterEditMode()"
```

Comment:
```suggestion
					@click="enterEditMode"
```

Reply:
First argument would be the click event object if we don't provide the `()`. Did this because `enterEditMode` is a method with arguments.